docs/sql-reference/10-sql-commands/00-ddl/07-aggregating-index/create-aggregating-index.md

```diff
@@ -16,10 +16,10 @@ Creates a new aggregating index in Databend.
 ## Syntax
 
 ```sql
-CREATE [SYNC] AGGREGATING INDEX <index_name> AS SELECT ...
+CREATE [ASYNC] AGGREGATING INDEX <index_name> AS SELECT ...
 ```
 
-- The `SYNC` parameter, when used during the creation of an aggregating index, enables automatic refresh, ensuring the index stays synchronized with the latest data updates.
+- The `ASYNC` parameter, when used during the creation of an aggregating index, need refresh manually. If `ASYNC` ignored, the aggregating indexes will refresh automatically, stay synchronized with the latest data updates.
```

Comment:
```suggestion
- The `ASYNC` parameter, when used during the creation of an aggregating index, means that the aggregating index will need to be refreshed manually in case of data updates. If `ASYNC` is skipped, the aggregating index will refresh automatically.
```